Tests/UnitTests/Core/Utilities/BoundingBoxTest.cpp

```diff
@@ -279,7 +293,7 @@ BOOST_AUTO_TEST_CASE(intersect_rays) {
     PlyVisualization3D<BoundingBoxScalar> ply;
 
     ray3.draw(ply);
-    std::ofstream os("ray3d.ply");
+    std::ofstream os(tmp("ray3d.ply"));
```

Comment:
```suggestion
    auto os = tmp("ray3d.ply");
```

would look more natural to me but feel free to ignore